### ISSUE TITLE
Missing "Enable background slideshows" under General skin settings

### ISSUE THREAD

**Issue by TheOne988** [realcopacetic/skin.copacetic#45]
In the latest version 1.1.7 the option to enable or disable the background slideshow is missing.

![image](https://github.com/realcopacetic/skin.copacetic/assets/36518440/535f845a-5223-4a47-b679-34d7e7b05979)

**FIX for the issue:**
The file `skin.copacetic/16x9/Content_Settings.xml` is missing a whole `Enable background fanart slideshow` control button. It should be returned as in previous versions:
```
<control type="radiobutton" id="1017">
	<description>Enable background fanart slideshow</description>
	<visible>ControlGroup(9).HasFocus(10)</visible>
	<label>$LOCALIZE[31425]</label>
	<onclick>Skin.ToggleSetting(Background_Disabled)</onclick>
	<selected>!Skin.HasSetting(Background_Disabled)</selected>
</control> 
```

**Comment by realcopacetic** [comment]
The reason I removed this was because it was getting very complicated supporting the various permeations of the light and dark themes so I just wanted to simplify because the design intention is that those screens would have a background image instead of being blank.

Do you want the ability to disable for aesthetic or performance reasons? I tried to ensure that the impact would be as minimal as possible. The background script only fetches artwork when a window with the slideshow is visible, otherwise it is paused.

**Comment by TheOne988** [comment]
Hi @realcopacetic,

I didn't think that the option was removed on purpose that why I reported it.
I think that performance is not the key factor because if you are using the art slideshow on the homepage it is not going to make some difference.
From a personal opinion I prefer to have some generic color or background when I'm in the menu browsing for some settings. The displayed art can sometimes be very bright or colorful and can interfere with the text from the menus.
I really don't know how complicated it is to support this because I have no experience in building Kodi skins.

Maybe it is even simpler if it doesn't show the art slideshow in the settings menus basically to hardcore some background/color. But as I said above I'm not an expert so I cannot know for sure if this suggestion is going to be simpler or not.

Anyway keep up the good work. I really like your skin that is why I report everything I find so I can use it without any hiccups 😊
